code/modules/shuttle/marine_dropship.dm

```diff
@@ -563,7 +569,7 @@
 	var/obj/docking_port/mobile/marine_dropship/M = SSshuttle.getShuttle(shuttleId)
 	if(!M)
 		return
-	if(!isxeno(usr) && M.hijack_state != HIJACK_STATE_NORMAL)
+	if(!isxeno(usr) && M.hijack_state == HIJACK_STATE_CALLED_DOWN)
 		to_chat(usr, "<span class='warning'>The shuttle isn't responding to commands.</span>")
```

Comment:
Why?

Reply:
Same as above. I suspect this code is not reacheable though